src/monitormanager.h

```diff
@@ -48,8 +49,11 @@ class MonitorManager : public IndexingObject<Monitor> {
     void unlock();
     std::string lock_number_changed();
 
+    void extractWindowStack(bool real_clients, std::function<void(Window)> addToStack);
+    void restack();
+
     int cur_monitor;
-    Stack* monitor_stack;
+    std::unique_ptr<PlainStack<Monitor*>> monitorStack_;
```

Comment:
Should this really keep being a public member, or are you planning on making it private later? (if it's the latter, I'd appreciate something like a `// TODO: make this private as soon as possible`

Reply:
This should become private soon; I've added the comment.